### PR TITLE
Update Remove-DnsServerResourceRecord.md

### PR DESCRIPTION
**Pull request by sowjanya365**
Issue # #2149 - Added a note for the parameter(-RecordData) to be required if user is running the command as a domain admin with delegated permissions.

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

cc 
@JohanFreelancer9 Copy editing is needed for this PR. Thanks
.
Peer review needed for this PR. Thanks.

**Comment by get-itips** [comment]
Hello @sowjanya365 
I tested this creating a regular user with DNS Admin permissions, and I could successfully remove a record without using `-RecordData`, maybe I misunderstood the scenario?

**Review by get-itips — COMMENTED** [review]
reviewed

File: docset/windows/dnsserver/Remove-DnsServerResourceRecord.md [hunk 1]
```diff
@@ -82,6 +82,8 @@ PS C:\> Remove-DnsServerResourceRecord -ZoneName "contoso.com" -RRType "A" -Name
 
 This command removes the A resource record from a zone named contoso.com that has the name Host01 and the IP address 10.17.1.41.
 
+If you are running this command as a domain admin with delegated permissions, "-RecordData" parameter is required.
```
Comment (JohanFreelancer9) [inline docset/windows/dnsserver/Remove-DnsServerResourceRecord.md:85]:
```suggestion
If you are running this command as a domain admin with delegated permissions, the "-RecordData" parameter is required. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by dariomws — COMMENTED** [review]
Agree with @get-itips.

@sowjanya365
How do you test it? How you define delegated permissions?

Maybe we can try adding permissions to a specific DNS zone, but I think what @get-itips tested should be enough.

Thanks!

**Comment by sowjanya365** [comment]
Thank you for your valuable inputs @get-itips @dariomws and @JohanFreelancer9! 

I'm sure, I might have missed something or any specific criteria that created this confusion. 

Could you please let me know if I can share this information with the user and close this PR? Thank you so much! 

FYI, @e0i! Thank you!

**Comment by dariomws** [comment]
@sowjanya365 I think so. What do you think @get-itips?

**Comment by get-itips** [comment]
> Thank you for your valuable inputs @get-itips @dariomws and @JohanFreelancer9!
> 
> I'm sure, I might have missed something or any specific criteria that created this confusion.
> 
> Could you please let me know if I can share this information with the user and close this PR? Thank you so much!
> 
> FYI, @e0i! Thank you!

Yes, but share that we tested this and could not reproduce to change the documentation.

**Pull request closed without merging**